src/stream.c

```diff
@@ -582,32 +582,34 @@ void LIBUSB_CALL _uvc_stream_callback(struct libusb_transfer *transfer) {
 
   switch (transfer->status) {
   case LIBUSB_TRANSFER_COMPLETED:
-    if (transfer->num_iso_packets == 0) {
-      /* This is a bulk mode transfer, so it just has one payload transfer */
-      _uvc_process_payload(strmh, transfer->buffer, transfer->actual_length);
-    } else {
-      /* This is an isochronous mode transfer, so each packet has a payload transfer */
-      int packet_id;
+    if ( strmh->running ) {
```

Comment:
Instead of adding this if-test and then indenting the following logic, you could instead just make this `if !strmh->running break;` (which will also solve the merge conflict I believe.)